python_env.md

```diff
@@ -4,33 +4,28 @@
 
 Install Python3: https://www.python.org/downloads/
 
-Install [pipenv](https://docs.pipenv.org/) to manage Python environments and pip dependencies:
-```
+Install [pipenv](https://docs.pipenv.org/) to manage Python environments and pip dependencies
+
+```sh
 pip3 install pipenv
 ```
-(You may need to source .bash_profile)
 
+(You may need to source .bash_profile)
 
 ## Set up Workspace Virtual Environment
 
 Run
 
-````
-./setup_python.sh
-````
+```sh
+`PIPENV_VENV_IN_PROJECT=1 pipenv install
+```
 
 That will create a Python 3.6 virtual environment under `.venv` to be used by Bazel, and install the development tools (`pylint`, `mypy`, `autopep8`, `rope`)
 
 On macOS you may get an error if you don't have the needed developer tools. You can use `xcode-select --install` to fix that.
 
 ## Add External Dependencies
 
-Libraries used in code should be listed in [third_party/py_requirements.txt](third_party/py_requirements.txt).
-
-Tools for development should be listed in [py_dev_requirements.txt](py_dev_requirements.txt).
+Libraries used in code should be listed in [setup.py](setup.py).
 
-To install the new dependencies without re-creating the Python virtual environment, run
-
-```
-./pipenv.sh install -r py_dev_requirements.txt
-```
\ No newline at end of file
+Tools for development should be listed in the [Pipfile](Pipfile).
```

Comment:
Maybe mention that they should run the installation again after changing the files.

Reply:
Done in 9d6eac5